.github/workflows/CI.yml

```diff
@@ -363,22 +392,67 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: 'aws-observability/aws-otel-collector-test-framework'
-          ref: 'v0.1.0'
-
-      - name: Run testing suite
+          path: testing-framework
+      
+      - name: Run testing suite on eks
         run: |
-          cd terraform/ecs && terraform init && terraform apply -auto-approve -var="ecs_launch_type=${{ matrix.type }}" -var="aoc_version=${{ needs.e2etest-preparation.outputs.version }}"
+          pwd=$PWD
+          cd testing-framework/terraform/ec2 && terraform init && terraform apply -auto-approve -var="testing_ami=${{ matrix.testing_ami }}" -var="aoc_version=${{ needs.e2etest-preparation.outputs.version }}" -var="otconfig_path=$pwd/e2etest/receiver/${{ matrix.testing_suite }}/otconfig.yaml" -var-file="$pwd/e2etest/receiver/${{ matrix.testing_suite }}/testing-suite.tfvar"
```

Comment:
q: `$pwd/e2etest/receiver/${{ matrix.testing_suite }}/otconfig.yaml` is just one of the config for the testing_suite. where does the iteration to test other folder/testing_suite?

Reply:
the matrix does the magic :)